site/app/controllers/admin/LateController.php

```diff
@@ -35,10 +35,15 @@ public function viewLateDays() {
      * @return Response
      */
     public function viewExtensions() {
+        $students = $this->core->getQueries()->getAllUsers();
+        $current_gid = isset($_COOKIE['exception_gid']) ? $_COOKIE['exception_gid'] : null;
+        $users = $this->core->getQueries()->getUsersWithExtensions($current_gid);
         return Response::WebOnlyResponse(
             new WebResponse(
                 ['admin', 'Extensions'],
                 'displayExtensions',
+                $students,
+                $users,
                 $this->core->getQueries()->getAllElectronicGradeablesIds()
             )
```

Comment:
The parameter order does not match the view.

Reply:
This has been fixed in my latest commit